src/ASM/LR/ASMu3D.h

```diff
@@ -339,6 +339,30 @@ class ASMu3D : public ASMunstruct, public ASM3D
   virtual bool evalSolution(Matrix& sField, const IntegrandBase& integrand,
                             const int* npe, char project = '\0') const;
 
+  //! \brief Transfers Gauss point variables from old basis to this patch.
+  //! \param[in] oldBasis The LR-spline basis to transfer from
+  //! \param[in] oldVars Gauss point variables associated with \a oldBasis
+  //! \param[out] newVars Gauss point variables associated with this patch.
+  //! \param[in] nGauss Number of Gauss points along a knot-span
+  virtual bool transferGaussPtVars(const LR::LRSpline* oldBasis,
+                                   const RealArray& oldVars, RealArray& newVars,
+                                   int nGauss) const;
+  //! \brief Transfers Gauss point variables from old basis to this patch.
+  //! \param[in] oldBasis The LR-spline basis to transfer from
+  //! \param[in] oldVars Gauss point variables associated with \a oldBasis
+  //! \param[out] newVars Gauss point variables associated with this patch.
+  //! \param[in] nGauss Number of Gauss points along a knot-span
+  virtual bool transferGaussPtVarsN(const LR::LRSpline* oldBasis,
+                                    const RealArray& oldVars, RealArray& newVars,
+                                    int nGauss) const;
+  using ASMunstruct::transferCntrlPtVars;
+  //! \brief Transfers control point variables from old basis to this patch.
+  //! \param[in] oldBasis The LR-spline basis to transfer from
+  //! \param[out] newVars Gauss point variables associated with this patch.
+  //! \param[in] nGauss Number of Gauss points along a knot-span
+  virtual bool transferCntrlPtVars(const LR::LRSpline* oldBasis,
+                                   RealArray& newVars, int nGauss) const;
+
```

Comment:
Same for 3D, rename oldBasis to what is used in the implementation file for the above three declarations

Reply:
done